pkg/serverless/daemon/daemon.go

```diff
@@ -185,6 +188,10 @@ func (d *Daemon) SetTraceAgent(traceAgent *trace.ServerlessTraceAgent) {
 	d.TraceAgent = traceAgent
 }
 
+func (d *Daemon) SetColdStartSpanCreator(creator *trace.ColdStartSpanCreator) {
```

Comment:
I guess you _technically_ don't need this method since the `ColdStartCreator` field is already exported.